GitHub_stuff.md

```diff
@@ -0,0 +1,7 @@
+# Github stuff we learned these two days
+
+First of all the MD tutorial is available: [Here](https://www.markdowntutorial.com)\
+Also, Praveen has been nice to provide us his tutorials and cheatsheets.
+
+<li>Github commands: [Here](https://blog.praveen.science/git-shortcuts/)
```

Comment:
Check the way how you need to use markdown. I told you to append `*` and a space in front of the lines.